src/server/service/debug.rs

```diff
@@ -175,10 +185,31 @@ impl debugpb_grpc::Debug for Service {
     fn scan_mvcc(
         &self,
         _: RpcContext,
-        _: ScanMvccRequest,
-        _: ServerStreamingSink<ScanMvccResponse>,
+        req: ScanMvccRequest,
+        sink: ServerStreamingSink<ScanMvccResponse>,
     ) {
-        unimplemented!()
+        let debugger = self.debugger.clone();
+        let future = move || {
+            let result: Result<(), ()> = future::result(debugger.scan_mvcc(req))
+                .map_err(|e| Service::error_to_grpc_error("scan_mvcc", e))
+                .and_then(|iter| {
+                    #[allow(deprecated)]
+                    stream::iter(iter)
+                        .map_err(|e| Service::error_to_grpc_error("scan_mvcc", e))
+                        .map(|(key, mvcc_info)| {
+                            let mut resp = ScanMvccResponse::new();
+                            resp.set_key(key);
+                            resp.set_info(mvcc_info);
+                            (resp, WriteFlags::default())
+                        })
+                        .forward(sink)
+                        .map(|_| ())
+                })
+                .map_err(|e| Service::on_grpc_error("scan_mvcc", &e))
+                .wait();
```

Comment:
Why wait?

Reply:
Because `DBIterator` is `!Send` so that we can't use it directly in `spawn`.